### DISCUSSION TITLE
Which repository hosting service should we add support for next?

### DISCUSSION THREAD

**Discussion by Ygnys** [GitLiveApp/GitLive#130] [Polls]
You can use GitLive in offline mode in any Git repository but our real-time features require signing into GitHub, GitLab, BitBucket or Azure DevOps.

You can vote for which repository hosting service should we add real-time support for next here, or add/upvote a comment below if your preferred service is missing from the poll.

**Comment by parda21** [comment]
I think Gitea should be integrated as a feature. 
We already have enough hosted variants. Please support self-hosted :)

greetings from Germany

**Comment by 6Alaska6Young6** [comment]
pls, notabug!

**Comment by inmativ** [comment]
I need gerrit integration.
Thanks!

**Comment by tmvkrpxl0** [comment]
I use codeberg, considering zig recently moved to there I think there's merit on supporting it too.
